modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -27,6 +27,26 @@ This illustration summarizes how each asset type (in green) appears in Exchange:
 
 image::ex2-exchange-assets.png[Illustration - How Assets appear in Exchange]
 
+== Asset Limits
+
+Exchange uses an asset limit to prevent accidental or intentional denial of service by uploading too many assets.
+
+The limit is 500 assets for master organizations with trial accounts, and 100 000 (one hundred thousand) assets for other master organizations.
+
+This asset count does not include soft deleted assets, hard deleted assets, or generated assets such as xref:to-deploy-using-rest-connect.adoc[Mule 3 and Mule 4 connectors generated automatically from APIs].
+
+When a master organization has reached 80% of its limit, Exchange shows a warning.
+
+When a master organization has reached its limit, Exchange shows an error. Also, using the *New asset* button on the home page or the *Add new version* button on the asset detail page shows an error explaining that the limit has been reached and no more assets can be added.
+
+Check the limit of your master organization with a CURL command like this. Replace `MASTER_ORGANIZATION_ID` with your master organization ID, and replace `ANYPOINT_TOKEN` with the authorization token with permissions for the master organization.
+
+----
+    curl -X GET \
```

Comment:
@mcampo , @LucasLudueno 
Shouldn't there be also an API that will display all the assets(with their types), that are counted towards the limit... in addition to the limit number?

Reply:
We can write a link point to this information if you want. To see the assets or an organization we have both GraphQL services or the XAPI. That was your question @kuntald ?

Reply:
Whatever API endpoint that will return the asset name & type for all the assets that count towards the limit, will be helpful here, for both XAPI & Graph.

Reply:
I would point to this documentation: https://docs.mulesoft.com/exchange/to-search-with-graph-api. There the users can find the way to search the assets for a master organization

Reply:
Can we please don't make the user jump to yet another documentation link? Can we just create a CURL command for the user to retrieve the total list of assets, using the API?

Reply:
Added the CURL command for getting asset lists.